pypesto/optimize/optimizer.py

```diff
@@ -558,3 +563,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, par_sigma0: float = 0.25, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+        self.par_sigma0 = 0.25
```

Comment:
No :) I mean not just set par_sigma0 to that fixed value internally, but use the argument

Reply:
Ah, you mean I should assign the par_sigma0 value declared in the __init__ to the self objective via self.par_sigma0 = par_sigma0 ?

Reply:
yes

Reply:
yes

Reply:
Done